database.js

```diff
@@ -0,0 +1,15 @@
+var knex = require('knex')({
+  client: 'mssql',
+  connection: {
+    host: '127.0.0.1',
```

Comment:
I would strongly suggest that you should create `.env` file and add it to your ignore list. All you would need then is requiring the variables you need.

Have a looks at this npm, it's the best way to secure your configuration:

https://github.com/motdotla/dotenv